src/app/Scenes/Home/Components/HomeHeader.tsx

```diff
@@ -3,15 +3,15 @@ import { ArtsyLogoIcon, Box, Flex, Spacer } from "palette"
 import { ActivityIndicator } from "./ActivityIndicator"
 
 export const HomeHeader: React.FC = () => {
-  const hasNotifications = GlobalStore.useAppState(
-    (state) => state.bottomTabs.sessionState.unreadCounts.unreadActivityPanelNotifications > 0
+  const shouldUnreadActivityPanelIndicator = GlobalStore.useAppState(
```

Comment:
what about changing the name to
```suggestion
  const shouldDisplayActivityPanelIndicator = GlobalStore.useAppState(
```